src/cls/rgw/cls_rgw.cc

```diff
@@ -2586,19 +2601,20 @@ static int list_plain_entries_help(cls_method_context_t hctx,
     try {
       decode(e, biter);
     } catch (ceph::buffer::error& err) {
-      CLS_LOG(0, "ERROR: %s: failed to decode buffer for plain bucket index entry \"%s\"",
-	      __func__, escape_str(iter.first).c_str());
+      CLS_LOG(0, "ERROR: %s: failed to decode buffer for plain bucket index entry %s",
+	      __func__, quote_escape(iter.first).c_str());
```

Comment:
what's wrong with quotes in the format strings? i prefer the existing version, especially when `quote_escape()` has runtime overhead